### ISSUE TITLE
Stun not work

### ISSUE THREAD

**Issue by somzoli** [jitsi/docker-jitsi-meet#1270]
After a few weeks ago the STUN configuration is not work. 
Tried google and jitsi stun servers but not worked. 

Log:
java.lang.IllegalArgumentException: No socket found for 172.25.0.5:10000/udp->100.121.102.50:63022/udp

The public IP is always 100.121.102.50.

Can you help with this? 

Thanks,
Z

**Comment by saghul** [comment]
This is not a real problem but a harmless error that unfortunately bubbles up.  Can we do something to squelch it and avoid confusion @bgrozev ?

**Comment by somzoli** [comment]
Other result: 
Pair failed: 172.18.0.4:10000/udp/host -> 100.121.102.50:59927/udp/host

The 100.121.102.50 ip address is not valid public IP. 

What can cause this?

**Comment by saghul** [comment]
That is a valid IP address.

**Comment by saghul** [comment]
The server didn't just come up with it. It must be the public IP of one of the connected clients.

**Comment by somzoli** [comment]
No, its not. My ip addres and the server ip address different from this. Where come from this IP?

**Comment by saghul** [comment]
I obviously don't know. Check your server access logs.

At any rate this has nothing to do with STUN not working.

**Comment by somzoli** [comment]
Can i change the JVB port to 10001?

**Comment by saghul** [comment]
Sure you can, just change the env var.

**Comment by somzoli** [comment]
I tried, but it looks that cause the problem. Can you test it with jvb port 10001?

**Comment by bgrozev** [comment]
What do you mean by "STUN doesn't work"? What is the expected behavior, what do you see instead?

**Comment by bgrozev** [comment]
> Can we do something to squelch it and avoid confusion @bgrozev ?

Yes, this change removes the stack trace from the logs (hasn't made it to jvb master yet):
https://github.com/jitsi/ice4j/pull/257
